src/main_chart.py

```diff
@@ -59,22 +62,26 @@ def get_main_chart_figure(
     collateral_token: str,
     debt_token: str,
 ) -> plotly.graph_objs.Figure:
+    # Define the AMMs and their color mappings
+    amms = ["jedi", "10k", "my", "sith"]
+    color_map = {"jedi": "#ECD662", "10k": "#4CA7D0", "my": "#76C1FA", "sith": "#FFA07A"}
```

Comment:
Use different shades of blue, to be consistent with the current version of the chart.